app/views/content/life-as-a-teacher/chemistry/_article.html.erb

```diff
@@ -0,0 +1,127 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach chemistry</h2>
+    <div class="inset">
+      <p>Chemistry underpins everything, so you'll be amazed at the questions your pupils will start asking. Leading hands-on lab experiments and creating engaging lessons will mean no two days are the same in the classroom.</p>
```

Comment:
```suggestion
      <p>Chemistry underpins everything, so you'll be amazed at the questions your pupils will start asking. Leading hands-on lab experiments and creating engaging lessons will mean no 2 days are the same in the classroom.</p>
```

Think our style is to put numbers - but this looks a bit odd to have `no 2` as you could read it as `number 2`!

Reply:
Yeah, it was originally a '2' but then changed it for that very reason!